src/components/Goods/Goods.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Goods.scss';
+import uuid from 'uuid';
+
+export const Goods = ({ goods }) => (
+  <ul className="list">
+    {goods.map(item => <li key={uuid.v4()} className="list__item">{item}</li>)}
```

Comment:
это так не будет работать, при каждом перерендере у тебя будет новый id, а нужно чтобы он всегда был постоянным
в этом случае 
```suggestion
    {goods.map(item => <li key={item} className="list__item">{item}</li>)}
```